src/services/GetMembers.jsx

```diff
@@ -1,17 +1,18 @@
 import axios from 'axios';
-import React from 'react'
+import React from 'react';
 import { toast } from 'react-toastify';
 
 const GetMembers = async () => {
-    try {
-        const res = await axios.get(
-          "http://localhost:5000/api/admin/members/21-22"
-        );
-        // toast.success("success")
-        return (res.data.members);
-      } catch (err) {
-        toast.error(err)
-      }
+  try {
+    const res = await axios.get(
+      "http://localhost:5000/api/admin/members/21-22"
```

Comment:
Define a global variable to store the base url and concat api endpoints for each api call